rust/xaynet-core/src/message/message.rs

```diff
@@ -478,9 +478,12 @@ pub struct Message {
     pub coordinator_pk: PublicEncryptKey,
     /// Wether this is a multipart message
     pub is_multipart: bool,
-    /// The type of message. This information is only partially
-    /// carried by the `payload` field, because in the case of a
-    /// multipart message, the payload is of type `Payload::Chunk`.
+    /// The type of message. This information is already partially
```

Comment:
```suggestion
    /// The type of message. This information is already
```

Reply:
Actually _partially_ is important here. When the payload is of type `Payload::Chunk` we don't know the actual type of the message

Reply:
Ah but I've put partially twice, I see.